brave-core/src/main/java/com/github/kristofa/brave/LocalTracer.java

```diff
@@ -68,6 +68,13 @@ static Builder builder(LocalTracer source) {
     }
 
     /**
+     * Clears current span.
+     */
+    public void clearCurrentSpan() {
```

Comment:
rather than opening a new tool for this job, probably better to direct folks to the existing one. Note that this was only recently noticed.

https://github.com/openzipkin/brave/commit/aca24d9d6ba6f1c00aef3594c3e14e5496296c13